### DISCUSSION TITLE
Can some please share the sample query file for MYSQL

### DISCUSSION THREAD

**Discussion by bgsunny-dev** [sqlc-dev/sqlc#1116] [Issue Triage] [answered]
I am very new to GO and as well as for RDS. I came across the sqlc to generate the code for database quires. Its very good package. All examples and documents talks only about using PostgreSQL. My project is based on the MYSQL. SQLC does mention that its support MYSQL, but couldn’t find query related documents for this. Can you please share sample quires documents.

**Comment by DuginKim** [comment, marked as answer]
https://github.com/kyleconroy/sqlc/tree/main/internal/endtoend/testdata

**Reply by bgsunny-dev** [reply]
thank you. This would help me a lot
